tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/NetActivitySdkTests.cs

```diff
@@ -73,6 +76,7 @@ public async Task SubmitsTraces()
                 ValidateIntegrationSpans(myServiceNameSpans, metadataSchemaVersion: "v0", expectedServiceName: "MyServiceName", isExternalSpan: false);
 
                 var settings = VerifyHelper.GetSpanVerifierSettings();
+                settings.AddRegexScrubber(_timeUnixNanoRegex, @"time_unix_nano"":<DateTimeOffset.Now>");
```

Comment:
Snapshot testing with span events is a bit challenging since we serialize timestamps in the string. I am wondering if we should normalize/ignore this field in the testagent (similar to what we do with span start time). To workaround this issue we mock `time` module. Another approach would be to deserialize the event string into json in the testagent. This will allow use to ignore/assert on span event fields. 

We have a similar issue with the trace_id and span_id fields in the `_dd.span_links` tag.

Reply:
I've decided to use a regex scrubber here because this allows me to test both a recent `DateTime.Now` property while also preserving a custom DateTimeOffset that was passed by the user, where I test the epoch and epoch+1. I'm not sure if you saw this in the snapshot files, but it helps me to know we can preserve the DateTimeOffset input